### DISCUSSION TITLE
Scripting is not enabled. The Open RDF Sesame Workbench application requires scripting to be enabled in order to work properly

### DISCUSSION THREAD

**Discussion by mdesalvo** [eclipse-rdf4j/rdf4j#3938] [General]
I have installed Tomcat9 on Ubuntu 22.04, deployed 4.0.1 war packages of RDF4J server and RDF4j workbench, I am able to access workbench to the initial page in Firefox. In this page I want to setup a new repository, but whatever I do it prints a quick red message which soonly disappears stating this error message:

"**Scripting is not enabled. The Open RDF Sesame Workbench application requires scripting to be enabled in order to work properly.**"

What am I doing wrong?

Regards,
Marco

**Comment by abrokenjester** [comment]
You say the message "soon disappears". What do you see after the message disappears? Can you continue to use the application?

PS clearly we need to update that error message.

**Comment by mdesalvo** [comment]
I remain in the page "http://localhost:8080/rdf4j-workbench/repositories/NONE/server" which is refreshed automatically causing the above error to disappear. I cannot perform any actions like "Create new repository", so the application simply cannot be used.
